CWE/events/East Turkestan Additions.txt

```diff
@@ -0,0 +1,1404 @@
+	picture = "turkestan_coalition"
+
+	is_triggered_only = yes
+
+
+	option = {
+		name = "Xinjiang is united once more!"
+		prestige = 10
+		inherit = ETS
+		change_capital = 1600
+		add_accepted_culture = uighur
+		1600 = {
+			change_province_name = "Urumqi"
+		}
+		any_pop = {
+			militancy = -3
+			consciousness = -3
+		}
+	}
+}
+
+#Reject - war
+country_event = {
+	id = 8013815
+	title = "East Turkestan Rejects Coalition Offer"
+	desc = EVTDESC8013815
+	picture = "ets_army"
+
+	is_triggered_only = yes
+
+
+	option = {
+		name = "Then we must ensure peace through force."
+		prestige = -10
+		any_pop = {
+			militancy = 3
+			consciousness = 3
+		}
+		war = {
+			target = ETS
+			attacker_goal = { casus_belli = civil_war }
+			defender_goal = { casus_belli = civil_war }
+			call_ally = no
+		}
+	}
+}
+
+#Proclaimation of an East Turkestani People's Republic
+country_event = {
+   id = 8013816
+   title = "East Turkestan Triumphant"
+   desc = EVTDESC8013816
+   picture = "ets_communist"
+
+	fire_only_once = yes
+
+	trigger = {
+		tag = ETS
+		war = no
+		OR = {
+			is_vassal = no
+			overlord = { OR = { government = proletarian_dictatorship government = proletarian_dictatorship1 }
+		} # I think this should ensure that if Tibet or some other nation puppets East Turkestan, it will not become communist, but being puppeted by a communist state will allow it to move along its event chain
+		XIJ = { exists = no }
+		NOT = { has_country_flag = ets_victory }
+	}
+
+	mean_time_to_happen = { months = 1 }
+
+	option = {
+		name = "Proclaim a East Turkestani People's Republic!"
+		government = proletarian_dictatorship
+		country_event = 800052
+		ruling_party_ideology = communist
+		prestige = 100
+		set_country_flag = ets_victory
+		1600 = {
+			change_province_name = "Urumqi"
+		}
+		any_pop = {
+			limit = { culture = uighur }
+			ideology = { factor = 0.10 value = communist }
+			militancy = -8
+			consciousness = -8
+		}
+
+		ai_chance = {
+			factor = 0.9
+		}
+	}
+
+	option = {
+		name = "We should moderate our ambitions."
+		government = socialist_democracy
+		country_event = 800054
+		ruling_party_ideology = communist
+		political_reform = universal_voting
+		political_reform = open_opinion
+		political_reform = limited_meetings
+		political_reform = elitist_service #Elitist
+		political_reform = parties_only_nominated
+		prestige = 100
+		set_country_flag = ets_victory
+		1600 = {
+			change_province_name = "Urumqi"
+		}
+		any_pop = {
+			limit = { culture = uighur }
+			ideology = { factor = 0.10 value = communist_social }
+			militancy = -8
+			consciousness = -8
+		}
+
+		ai_chance = {
+			factor = 0.1
+		}
+	}
+}
+
+
+#Sovietization of East Turkestan
+country_event = {
+   id = 8013817
+   title = "Joining the Soviet Bloc"
+   desc = EVTDESC8013817
+   picture = "nwo2_east_turkestan"
+
+	fire_only_once = yes
+
+	trigger = {
+		tag = ETS
+		war = no
+		is_vassal = no
+
+		RUS = { government = proletarian_dictatorship }

```

Comment:
Need stricter checks for this event

RUS = { exists = yes government = proletarian_dictatorship is_greater_power = yes }